include/click/clp.h

```diff
@@ -88,6 +88,8 @@ struct Clp_Option {
 #define Clp_ValDouble		8	/**< @brief Option value is a
 					     double.
 		Accepts a real number as defined by strtod(). */
+#define Clp_ValStringDashTerimnated	9	/**< @brief Option value is a string
```

Comment:
@kohler could you validate the way I'm adding "double dash terminated arguments" in clp.{c,hh} ? I'm not sure it's the right method.

Note that DPDK parses commands until it encounters a double dash, it was why I chose the double dash thing. It is more DPDK-like.